operator-common/src/main/java/io/strimzi/operator/common/model/Ca.java

```diff
@@ -530,7 +527,7 @@ public void createRenewOrReplace(
         if (!generateCa) {
             certData = caCertSecret.getData();
             keyData = singletonMap(CA_KEY, caKeySecret.getData().get(CA_KEY));
-            renewalType = hasCaCertGenerationChanged(existingServerSecrets) ? RenewalType.REPLACE_KEY : RenewalType.NOOP;
+            renewalType = RenewalType.NOOP; // User is managing CA
```

Comment:
How does this impact the other parts of the reconciliation? E.g. generating server / user certs based on the new custom CA etc.?

Reply:
In short, it does not impact it.

In detail:
In `ClusterCa` in `[maybeCopyOrGenerateCerts](https://github.com/strimzi/strimzi-kafka-operator/blob/main/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ClusterCa.java#L308)` we check two things `this.certRenewed()` (which checks the value of `renewalType`) and `hasCaCertGenerationChanged(secret)`. If either are true we renew the certificates.

With this change if a user managed CA public cert is renewed `this.certRenewed()` now returns false instead of true, however `hasCaCertGenerationChanged(secret)` still returns true so the certs are renewed.

We do similar checks everywhere related to determining whether to renew certificates.